specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2452,6 +2455,11 @@
         "allowSharedKeyAccess": {
           "type": "boolean",
           "description": "Indicates whether the storage account permits requests to be authorized with the account access key via Shared Key. If false, then all requests, including shared access signatures, must be authorized with Azure Active Directory (Azure AD). The default value is null, which is equivalent to true."
+        },
+        "isNfsV3Enabled": {
+          "type": "boolean",
+          "x-ms-client-name": "EnableNfsV3",
+          "description": "NFS 3.0 protocol support enabled if sets to true."
```

Comment:
Per my test, "EnableNfsV3" can only be set to true, when create account with following parameters.
If they are necessary, please add the limitation to the description of the parameter, to make user easily to use the parameter.
1. isHnsEnabled = trure
2. supportsHttpsTrafficOnly = false
3. create account with networkAcls.virtualNetworkRules (the rule need be in same location with the account)
4. I test with: Kind: BlockblobStorage; SkuName: Premium_LRS